iml-gui/crate/src/page/user.rs

```diff
@@ -81,6 +83,14 @@ pub struct Model {
 }
 
 impl Model {
+    pub fn title(&self) -> String {
+        match (self.user.first_name.as_str(), self.user.last_name.as_str()) {
+            ("", "") => self.user.username.clone(),
+            ("", l) => l.into(),
+            (f, "") => f.into(),
+            (f, l) => format!("{} {}", f, l),
```

Comment:
Nice!

Reply:
I am just standing on the shoulders of giants ;)